src/index.js

```diff
@@ -8,42 +8,44 @@ const feedWorker = require('./feed-worker');
 
 /**
  * Process a string into a list of Objects, each with a feed URL
- * @param {String} lines 
+ * @param {String} lines
  */
 function processFeedUrls(lines) {
   return lines
     // Divide the file into separate lines
     .split(/\r?\n/)
-    // Basic filtering to remove any ines that don't look like a feed URL 
-    .filter(line => line.startsWith('http'))
+    // Basic filtering to remove any ines that don't look like a feed URL
+    .filter((line) => line.startsWith('http'))
     // Convert this into an Object of the form expected by our queue
-    .map(url => ({ url }));
+    .map((url) => ({ url }));
 }
 
 /**
  * Adds feed URL jobs to the feed queue for processing
- * @param {Array[Object]} feedJobs - list of feed URL jobs to be processed 
+ * @param {Array[Object]} feedJobs - list of feed URL jobs to be processed
  */
-async function enqueueFeedJobs(feedJobs) {
-  for(let feedJob of feedJobs) {
+function enqueueFeedJobs(feedJobs) {
+  feedJobs.forEach(async (feedJob) => {
     console.log(`Enqueuing Job - ${feedJob.url}`);
     await feedQueue.add(feedJob);
-  }
+  });
 }
 
 /**
  * For now, do something simple and just read a few feeds from a text file
  */
 fs.readFile('feeds.txt', 'utf8', (err, lines) => {
-  if(err) {
+  if (err) {
     console.error('unable to read initial list of feeds', err.message);
     return process.exit(-1);
   }
 
   // Process this text file into a list of URL jobs, and enqueue for download
   const feedJobs = processFeedUrls(lines);
-  enqueueFeedJobs(feedJobs);  
+  enqueueFeedJobs(feedJobs);
 
   // Start working on the queue
   feedWorker.start();
+  //Returning true if no other errors were encountered
```

Comment:
I don't understand this addition.  Nothing is consuming this return value, it's just a callback.  If airbnb is requiring this, we may want to override that.

Reply:
It was airbnb requiring it, yes.

Reply:
Oh, actually it's standard ESLint.

Viewable here: https://eslint.org/docs/rules/consistent-return

Reply:
I believe it is throwing an error because of the "return process.exit(-1);"

There is only one path to that return, which is counting as odd to ESLint.

Reply:
This is the only thing unresolved, as we now know why it is erroring and I thought could use some further commentary. I can still disable the rule if you wish.

Reply:
What if you modify my code so it does this:

```js
...
    process.exit(-1);
    return;
}

...
```

And don't do a return at the end.  Is it happy with that?  The extra return here might be making the linter happy, but the code doesn't make as much sense.  Let me know, and maybe we'll disable this rule.

Reply:
It is happy with that, as it technically returns no value.

Reply:
Adjusted and updated.